src/porepy/numerics/fv/biot.py

```diff
@@ -199,12 +201,10 @@ def assemble_matrix(self, g, data):
         # Matrix for left hand side
         A_biot = sps.bmat(
             [
-                [A_mech, matrices_m["grad_p"] * biot_alpha],
+                [A_mech, grad_p],
```

Comment:
Don't we want to keep the biot alpha factor on grad_p?

Reply:
in the new discretization alpha is already inside grad_p

Reply:
Ok, good!